content/projects/gsoc/2020/projects/github-checks.adoc

```diff
@@ -24,4 +23,48 @@ links:
   idea: /projects/gsoc/2020/project-ideas/github-checks
 ---
 
-// TODO: add details here, remove `showGoogleDoc: true`
+The link:https://developer.github.com/v3/checks/[GitHub Checks API] allows developers to report the CI integrations’ detail information rather than the binary pass/fail build status on GitHub pages. This project is about implementing this API as a new Jenkins plugin. By consuming this API, other plugins can easily create GitHub checks. Thus, any information during the Jenkins CI process like warnings, summaries, and durations can be directly shown on GitHub pages.
```

Comment:
Best practice: add line breaks so that each sentence starts on a new line

Reply:
Also keep in mind that "Jenkins CI" is a deprecated expression. Better to just use "Jenkins"